dist/ng-wig.js

```diff
@@ -148,14 +154,21 @@ angular.module('ngWig').component('ngWig', {
         return;
       }
 
-      var pasteContent = (event.originalEvent || event).clipboardData.getData('text/plain');
+      var pasteContent = void 0;
+      if (window.clipboardData && window.clipboardData.getData) {
```

Comment:
actually, that's not a change of mine. It just appeared in dist when I did the build by mistake. Probably the change was introduced in a previous commit and no build was run since then.